test/common/testlib.py

```diff
@@ -2419,6 +2419,8 @@ def test_main(options=None, suite=None, attachments=None, **kwargs):
                         default=None, help="Run this test against an already running machine")
     parser.add_argument('--browser', metavar="hostname[:port]", dest="browser",
                         default=None, help="When using --machine, use this cockpit web address")
+    parser.add_argument('-c', action='store_true',
+                        help="Equivalent to '--machine 127.0.0.2:2201 --browser 127.0.0.2:9091'")
```

Comment:
Unrelated, and most probably debugging leftover. Let's rather keep this local (I have a `$RUNC` variable for that)

Reply:
Removed.